ui/apps/platform/src/Containers/SystemConfig/Form/SystemConfigForm.tsx

```diff
@@ -333,6 +333,26 @@ const SystemConfigForm = ({
                         />
                     </FormGroup>
                 </GridItem>
+                <GridItem>
+                    <FormGroup
+                        label="Prepared downloadable vulnerability reports retention days"
+                        isRequired
+                        fieldId="privateConfig.reportRetentionConfig.downloadableReportRetentionDays"
+                    >
```

Comment:
nit: These next-pruning fields probably need `helperText` that explains 0 means something different for them.

Don't hold the PR for it, but worth adding later.

Reply:
Good callout. I was going to bring this up in the UX sync this week. Seems a little strange that the logic is different when 0 is used without any explanation